data_management/data_factory/models/marts/1_agregation_geographique/faits_indicateur_national.sql

```diff
@@ -10,10 +10,9 @@ WITH faits_indicateur__national_user_input as (
     FROM {{ ref("faits_indicateur_deduplique") }} fi_deduplique
     LEFT JOIN {{ ref("stg_ppg_metadata__parametrage_indicateurs") }} parametrage_indicateurs
         ON fi_deduplique.indicateur_id = parametrage_indicateurs.indicateur_id
-    WHERE zone_type = 'NAT'
-        AND (parametrage_indicateurs.vi_nat_from = 'user_input' AND fi_deduplique.type_mesure = 'vi')
-        OR (parametrage_indicateurs.va_nat_from = 'user_input' AND fi_deduplique.type_mesure = 'va')
-        OR (parametrage_indicateurs.vc_nat_from = 'user_input' AND fi_deduplique.type_mesure = 'vc')
+    WHERE (zone_type = 'NAT' AND (parametrage_indicateurs.vi_nat_from = 'user_input' AND fi_deduplique.type_mesure = 'vi'))
+        OR (zone_type = 'NAT' AND (parametrage_indicateurs.va_nat_from = 'user_input' AND fi_deduplique.type_mesure = 'va'))
+        OR (zone_type = 'NAT' AND (parametrage_indicateurs.vc_nat_from = 'user_input' AND fi_deduplique.type_mesure = 'vc'))
```

Comment:
Faut qu’on revoit ensemble les parenthèses, j’ai un doute (cf mon message) je pense que c’est bon (voir sûrement factorisable avec zone_type=‘FR’ AND ( (conditions 1) OR (conditions 2) OR (conditions 3))